polaris-server.yml

```diff
@@ -167,3 +167,5 @@ logging:
       archivedLogFilenamePattern: ./logs/polaris-%d.log.gz
       # The maximum number of log files to archive.
       archivedFileCount: 14
+
+maxDocumentBytes: 0 # Don't set a limit
```

Comment:
1. This comment would ideally not be inline
2. Do we really want to call this `maxDocumentBytes`? What is a "Document" in the context of a Polaris instance?
3. Whatever we call it, we might put a small comment explaining what this is

Reply:
Renamed to maxRequestBodyBytes and updated/moved the comment.